src/executors/node_redis.yml

```diff
@@ -0,0 +1,14 @@
+---
+description: >
+  Executor for node.js jobs with redis image
+docker:
+  - image: 'cimg/node:<<parameters.node_tag>>'
+  - image: 'circleci/redis:<<parameters.redis_tag>>'
+working_directory: ~/app
+parameters:
+  node_tag:
+    default: "14.16-browsers"
```

Comment:
The typescript template project, which could make use of this executor, uses 14.17 here

Reply:
agreed, but wanted to get it initally to match the `node` executor

Reply:
could bump the hmpps/node executor too.

Reply:
so that if people grab both for their projects and reference both without tags they don't get a version mismatch

Reply:
it's all annoying as there's a load of different variations too - using postgres, mssql, cypress... etc..

Then if you do start using the default version specified here then its really easy for this to silently mismatch the version specified in the docker image.. why can't we just have nice things? 😄

Reply:
agreed - I'm tempted to remove the default version here as it always gets out of date and a nightmare then to change.  People should then explicitly have to specify it